### PR TITLE
Feature/devops

### PR DESCRIPTION
**Pull request by MrRTi**

### CODE REVIEW

**Comment by artempartos** [comment]
squash commits please

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - DOCKER_COMPOSE_VERSION=1.22.0
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - docker-compose build
+  - docker-compose run --rm web /bin/sh -c "bundle install; RAILS_ENV=test  bundle exec rails db:create db:migrate"
+
+language: ruby
```
Comment (artempartos) [inline .travis.yml]: why do you need ruby and rvm once you do it in docker?

File: Gemfile [hunk 2]
```diff
@@ -45,6 +43,11 @@ gem "therubyracer"
 gem "less-rails" #Sprockets (what Rails 3.1 uses for its asset pipeline) supports LESS
 gem "twitter-bootstrap-rails"
 gem 'jquery-rails'
+gem 'simplecov', require: false, group: :test
+gem 'coveralls', require: false
+gem 'rollbar'
+gem 'rails-monitoring'
```
Comment (artempartos) [inline Gemfile]: do we have a sidekiq here? I suppose rails-monitoring is for monitoring sidekiq

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
-# Tutorial_project
+# Tutorial_project [![Build Status](https://travis-ci.org/MrRTi/tutorial_01.svg?branch=master)](https://travis-ci.org/MrRTi/tutorial_01) [![Coverage Status](https://coveralls.io/repos/github/MrRTi/tutorial_01/badge.svg?branch=master)](https://coveralls.io/github/MrRTi/tutorial_01?branch=master)
```
Comment (artempartos) [inline README.md]: add develop branch  as well

File: Gemfile (hunk 2 above)
Comment (MrRTi) [reply Gemfile]: removed the rails-monitoring from project

**Pull request merged**